lib/networkPacks/PacksForClient.h

```diff
@@ -1064,17 +1064,25 @@ struct DLL_LINKAGE BulkMoveArtifacts : CArtifactOperationPack
 
 	ObjectInstanceID srcArtHolder;
 	ObjectInstanceID dstArtHolder;
+	std::optional<SlotID> srcCreature;
+	std::optional<SlotID> dstCreature;
```

Comment:
Alternatively we can add SlotID::NONE constant that will be used for cases like this one, instead of optional. 
Up to you.

Reply:
In ArtifactLocation is also used std::optional<SlotID>. Therefore, it is more convenient to use the same.
In general, there is no difference. Both are usable.